BehaviorTreeEditor.cs

```diff
@@ -1,18 +1,35 @@
-﻿
-#if UNITY_EDITOR
+﻿#if UNITY_EDITOR
 using UnityEngine;
 using UnityEditor;
 using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
 
+namespace BehaviorTree
+{
 public class BehaviorTreeEditor : EditorWindow
```

Comment:
I'm going to have to do this in the next pull request, but I'm fine with the change.